modules/passkey/test/userStories/rotatePasskeyOwner.ts

```diff
@@ -0,0 +1,181 @@
+// Import necessary dependencies from chai, hardhat, @safe-global/safe-4337, webauthn
+import { expect } from 'chai'
+import { deployments, ethers } from 'hardhat'
+import { WebAuthnCredentials, decodePublicKey } from '../utils/webauthn'
+import { Safe4337Module } from '@safe-global/safe-4337/typechain-types/contracts/Safe4337Module'
+import {
+  buildSafeUserOpTransaction,
+  buildPackedUserOperationFromSafeUserOperation,
+  signSafeOp,
+} from '@safe-global/safe-4337/src/utils/userOp'
+import { buildSignatureBytes } from '@safe-global/safe-4337/src/utils/execution'
+import { chainId } from '@safe-global/safe-4337/test/utils/encoding'
+import { Log } from 'hardhat-deploy/types'
+
+/**
+ * User story: Rotate passkey owner
+ * In the setup, deploy a Safe with an EOA and a passkey signer as owners with threshold 1.
+ * In the test, the EOA executes a userOp to swap the passkey signer with a new passkey signer.
+ *
+ * The flow can be summarized as follows:
+ * Step 1: Setup the contracts
+ * Step 2: Create a userOp to swap the passkey signer, sign it with EOA wallet
+ * Step 3: Execute the userOp to swap the passkey signer
+ */
+describe('Rotate passkey owner [@User story]', () => {
+  // Create a fixture to setup the contracts and signer(s)
+  const setupTests = deployments.createFixture(async ({ deployments }) => {
+    const { EntryPoint, Safe4337Module, SafeProxyFactory, SafeModuleSetup, SafeL2, FCLP256Verifier } = await deployments.run()
+
+    // EOA which will be the owner of the Safe
+    const [user] = await ethers.getSigners()
+
+    const entryPoint = await ethers.getContractAt('IEntryPoint', EntryPoint.address)
+    const module = (await ethers.getContractAt(Safe4337Module.abi, Safe4337Module.address)) as unknown as Safe4337Module
```

Comment:
```suggestion
    const module = await ethers.getContractAt(Safe4337Module.abi, Safe4337Module.address)
```

Reply:
I think this may be needed for typing to work correctly. Another alternative is to include `Safe4337Module` in the `Dependencies.sol` file so that you can:

```suggestion
    const module = await ethers.getContractAt('Safe4337Module', Safe4337Module.address)
```

Which will be correctly typed.